dotnet/src/SemanticKernel.Core/SemanticKernel.Core.csproj

```
@@ -34,6 +34,7 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.AI" />
```

`SK.Abstractions` already depends on `MEAI.Abstractions`, this and the NU5104 is not needed.

```suggestion
    <PackageReference Include="Microsoft.Extensions.AI" />
```

---

`SK.Abstractions` only depends on `Microsoft.Extensions.AI.Abstractions`, not `Microsoft.Extensions.AI`, that said it doesn't look like I need it anymore.  Not sure why I needed it before, but it's working now without it.  I'll remove.